docs/handbook/getting-started.md

```diff
@@ -38,6 +38,32 @@ This collection of documents is aimed at providing a comprehensive source of gui
 While our aim is to address as many facets of OpenIndiana use and administration as possible, some topics are simply too complex and beyond the scope of an introductory end user handbook.
 In such cases external sources of information will be provided in the form of references to other web sites, man pages, or printed books.
 
+## Conventions
+
+### Commandlines
+
+Commandlines starting with a # character means that they should be run as root.
```

Comment:
"Command lines" or "command-lines" could work instead. Examples is another alternative.